packages/components/organisms/f-checkout/src/components/Address.vue

```diff
@@ -91,23 +96,23 @@ export default {
         ...mapState('checkout', [
             'fulfilment'
         ]),
+
         /*
         * Validation methods return true if the validation conditions
         * have not been met and the field has been `touched` by a user.
         * The $dirty boolean changes to true when the user has focused/lost
         * focus on the input field.
         */
-
         isAddressLine1Empty () {
-            return this.isFieldEmpty('line1');
+            return this.isFieldEmpty('address', 'line1');
```

Comment:
Does the casing matter here? In some places it's "Address"

Reply:
Have added suggested `VALIDATIONS` to constants file